### ISSUE TITLE
`draw_text` needs more api docs

### ISSUE THREAD

**Issue by einarf** [pythonarcade/arcade#985]
Loooking at the api docs it's not clear what options and combinations are avaialble for many of the parameters. The user needs to look at examples or guess from pyglet docs.

https://github.com/pythonarcade/arcade/blob/82d8085bd446ac42c634f56a4d9b8d97ac01b417/arcade/text_pyglet.py#L73-L107

**Comment by pushfoo** [comment]
I'm working on this at the moment.

@pvcraven Was `wrap_lines` supposed to be added at some point? Git history says you added that param to the docstring, but it's currently neither in the function signature nor body. I also don't see it in the pyglet doc for `pyglet.text.Label`

**Comment by pushfoo** [comment]
~~Given that #992 renders the Text class unusable, should I polish and submit what I have for `draw_text`? it seemed like there was a bugfix release coming up shortly from what I read in discord earlier.~~

**Comment by pushfoo** [comment]
I'll finish this after #992 is reviewed and merged, and document `arcade.Text` in it as well.

**Comment by pushfoo** [comment]
~~Waiting for resolution on #1005 before proceeding. I have tables and some other documentation formatting mostly figured out, but I need a clear answer on the sphinx config and docstrings standards for the project before proceeding.~~

Per discord discussion, the project standard is that classes should use the top level class docstring for both constructor arguments and attributes. I'll work on this more later. Much of the work of learning how to format with RST and turning some examples into tables is already done.
